### PR TITLE
Jeremy/host-portal-rebrand

### PR DESCRIPTION
**Pull request by jerminatorhits**
## Description
Why did you write this code?
Rebrand host/bookings

## How to Test
- [ ] Visit host/
- [ ] Verify functionality of pages/tabs
- [ ] If logged in as a user with a listing, the header button should read "Host Profile" and take you to host portal '/host'
- [ ] If logged in as a user without a listing or not logged in, it should read "Become a Host" and take you to signup pages
- [ ] Test stripe routes on host/payment

Which devices did you test on?
- [ ] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/legacy/routes/Host/HostBookingCard/HostBookingCard.tsx [hunk 1]
```diff
@@ -14,6 +11,9 @@ import {
 } from 'utils/bookingsDisplayStatus';
 import { currencyToDisplay } from 'utils/currencyToDisplay';
 import { formatDateRange } from 'utils/formatDate';
+import { Card, Col, Row } from 'reactstrap';
+import Loading from '../../../shared/loading/Loading';
```
Comment (kcvan) [inline src/legacy/routes/Host/HostBookingCard/HostBookingCard.tsx]: can we use the `/legacy/shared/etc` import here? will be easer to change all occurrences later

File: src/legacy/work/Header/Header.tsx [hunk 2]
```diff
@@ -68,33 +71,58 @@ const DetailedHeader = () => {
...
+                                  {item.header}
+                                </NavLink>
+                              </NavItem>
+                            ))}
+                          </>
+                        );
+                      }}
+                    </Query>
+                  )
+                }
```
Comment (jerminatorhits) [inline src/legacy/work/Header/Header.tsx]: All this just to dynamically render the Host Portal button name on the header O_O

File: src/legacy/routes/Host/HostBookingCard/HostBookingCard.tsx (hunk 1 above)
Comment (jerminatorhits) [reply src/legacy/routes/Host/HostBookingCard/HostBookingCard.tsx]: 👍

File: src/legacy/work/Header/Header.tsx (hunk 2 above)
Comment (kcvan) [reply src/legacy/work/Header/Header.tsx]: 🙈

File: src/legacy/work/Header/Header.tsx [hunk 3]
```diff
@@ -68,33 +71,58 @@ const DetailedHeader = () => {
...
+                        return (
+                          <>
+                            {beeUser.listingCount > 0
+                              ? <Link to={HOST_PORTAL_LINK} className="mb-4 mb-md-0 mr-md-4 w-100 w-md-auto btn btn-outline-primary">
+                                  Host Profile
+                                </Link>
+                              : <Link to={HOST_INTEREST_LINK} className="mb-4 mb-md-0 mr-md-4 w-100 w-md-auto btn btn-outline-primary">
+                                  Become a Host
+                                </Link>
+                            }
```
Comment (woeltjen) [inline src/legacy/work/Header/Header.tsx]: Looks like there's a small opportunity for deduplication here with a

    const isHost = beeUser.listingCount > 0;

And then a

    <Link to={isHost ? HOST_PORTAL_LINK : HOST_INTEREST_LINK} className="letter-hyphen-number-party">
      {isHost ? 'Host Profile' : 'Become a Host'}
    </Link>

File: src/legacy/work/Header/Header.tsx (hunk 2 above)
Comment (woeltjen) [reply src/legacy/work/Header/Header.tsx]: Worth moving some of this into its own component? The nesting gets pretty deep, but 142 LOC still seems reasonable for the Header as a whole, so I'm neutral

File: src/legacy/work/Header/Header.tsx (hunk 3 above)
Comment (jerminatorhits) [reply src/legacy/work/Header/Header.tsx]: 🙏

File: src/legacy/work/Header/Header.tsx (hunk 2 above)
Comment (jerminatorhits) [reply src/legacy/work/Header/Header.tsx]: Hmmm nesting does get pretty deep. I don't anticipate the Header changing very much from this point so I think it'd be good to keep logic local for now.

**Pull request merged**